src/main/io/osd.c

```diff
@@ -1472,39 +1472,31 @@ static bool osdDrawSingleElement(uint8_t item)
         {
             char *p = "ACRO";
 
-            if (isAirmodeActive()) {
-                p = "AIR ";
-            }
-
-            if (FLIGHT_MODE(FAILSAFE_MODE)) {
+            if (FLIGHT_MODE(FAILSAFE_MODE))
                 p = "!FS!";
-            } else if (FLIGHT_MODE(MANUAL_MODE)) {
+            else if (FLIGHT_MODE(MANUAL_MODE))
                 p = "MANU";
-            } else if (FLIGHT_MODE(NAV_RTH_MODE)) {
+            else if (FLIGHT_MODE(NAV_RTH_MODE))
                 p = "RTH ";
-            } else if (FLIGHT_MODE(NAV_POSHOLD_MODE)) {
-                if (FLIGHT_MODE(NAV_ALTHOLD_MODE)) {
-                    // 3D HOLD
-                    p = "HOLD";
-                } else {
-                    p = " PH ";
-                }
-            } else if (FLIGHT_MODE(NAV_CRUISE_MODE) && FLIGHT_MODE(NAV_ALTHOLD_MODE)) {
+            else if (FLIGHT_MODE(NAV_POSHOLD_MODE))
+                p = "HOLD";
+            else if (FLIGHT_MODE(NAV_CRUISE_MODE) && FLIGHT_MODE(NAV_ALTHOLD_MODE))
                 p = "3CRS";
-            } else if (FLIGHT_MODE(NAV_CRUISE_MODE)) {
+            else if (FLIGHT_MODE(NAV_CRUISE_MODE))
                 p = "CRS ";
-            } else if (FLIGHT_MODE(NAV_ALTHOLD_MODE) && navigationRequiresAngleMode()) {
+            else if (FLIGHT_MODE(NAV_ALTHOLD_MODE) && navigationRequiresAngleMode()) {
```

Comment:
And here we have the reason why you should always add braces. Otherwise it leads to inconsistent and difficult to read code.